cpp/src/Ice/EndpointFactoryManager.cpp

```diff
@@ -63,18 +63,18 @@ IceInternal::EndpointFactoryManager::get(int16_t type) const
 }
 
 EndpointIPtr
-IceInternal::EndpointFactoryManager::create(const string& str, bool oaEndpoint) const
+IceInternal::EndpointFactoryManager::create(string_view str, bool oaEndpoint) const
 {
     vector<string> v;
     bool b = IceInternal::splitString(str, " \t\n\r", v);
     if (!b)
     {
-        throw ParseException(__FILE__, __LINE__, "mismatched quotes in endpoint '" + str + "'");
+        throw ParseException(__FILE__, __LINE__, "mismatched quotes in endpoint '" + string{str} + "'");
```

Comment:
Would be a bit nicer to use `ostringstream` to avoid creating the string from the string_view.

Reply:
Very minor since it's in an exception.